beacon-chain/rpc/validator_server.go

```diff
@@ -43,8 +43,8 @@ func (vs *ValidatorServer) ValidatorEpochAssignments(
 	ctx context.Context,
 	req *pb.ValidatorEpochAssignmentsRequest,
 ) (*pb.ValidatorEpochAssignmentsResponse, error) {
-	if len(req.PublicKey) != 48 {
```

Comment:
Why change this? Didn't we have panics where invalid public keys were sent?

Reply:
IMO, this needs to be fixed to the expectation size. If it's 96 now then it should be 96.

Reply:
pub key size should really become its own variable if it's referenced in multiple places

Reply:
yes, it can be a params constant